src/components/dialog/DefaultDialog.vue

```diff
@@ -36,8 +36,7 @@ const show = ref(false);
 let dialogEventSubscription;
 
 /**
- *
- * @param event
+ * @param {object} event - Event.
  */
 function onDialogEvent(event) {
```

Comment:
check if param `event` can be deconstructed.

Reply:
done